### PR TITLE
Removed redundant code from the edit view of user

### PR DESCRIPTION
**Pull request by SidharthBansal**
Fixes part of #4269 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: app/views/users/_form.html.erb [hunk 1]
```diff
@@ -3,20 +3,7 @@
 <% if f.error_messages != "" %><div class="alert alert-danger"><%= f.error_messages %></div><% end %>
```
Comment (SidharthBansal) [inline app/views/users/_form.html.erb]: For the modals to work, we need to supply the url in the form. So we created a new _form2 partial for the user creation view and used existing _form partial for the edit user view. So removing these redundant code now.

**Comment by SidharthBansal** [comment]
image
image

image

image

image

Everything is working correct. All tests are passing.

**Comment by SidharthBansal** [comment]
@jywarren can we merge this?

**Comment by jywarren** [comment]
If it looks good to you!!!

**Comment by SidharthBansal** [comment]
Can you please review and suggest any changes ?
Thanks

**Comment by jywarren** [comment]
Ok, I will try to as soon as possible.

**Comment by SidharthBansal** [comment]
@jywarren just I want you to see the changes going on in these files so that you can assist other upcoming folks. These changes are minor so I merged them.
For the sign up modal to work we need to supply the route in the form as parameters. So we have to divide form partial into create_form and edit_form partial for separate actions.
Thanks

**Comment by jywarren** [comment]
This looks perfect, although if you could be sure the 'mobile view' works well too, that'd be great. Then +1 to merging!

**Comment by jywarren** [comment]
Oh ok - just saw your update. This is great. Thanks!!!

**Pull request merged**